my-vue3-app/src/store/reports/actions.ts

```diff
@@ -37,6 +37,7 @@ export default {
   async [FETCH_REPORTS](
     { commit, state }: ActionContext<State, unknown>,
     desired: Omit<Pagination, 'viewIds'|'ids'>,
+    // filters: Pick<Report, 'nick'>
```

Comment:
see `// filter` [at line 18](https://github.com/tomfun/ero-like/blob/6c4fc48168de60dbf6face44eac61df8d02730af/my-vue3-app/src/store/reports/actions.ts#L18)

Reply:
you should implement filtering. and clearing: if (oldFilters === newFilters) do guessinge else fuckIt